src/components/App.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+
+import Page from './Page';
+
+export default function App() {
+  const [state, setState] = useState({
+    todos: [],
+    input: '',
+  });
+
+  const { todos, input } = state;
+
+  const handleChangeInput = (event) => {
+    const currentInput = event.target.value;
+    setState({ todos, input: currentInput });
+  };
+
+  const handleClickAddTodo = (event) => {
+    event.preventDefault();
+    setState({ todos: [...todos, { id: todos.length + 1, title: input }], input: '' });
```

Comment:
```suggestion
setState({ 
    todos: [
        ...todos,
        {  id: todos.length + 1, title: input }
    ],
    input: ''
});
```

이렇게 한번 제안해봤는데 비교했을 떄 어떠하신가요?

작성하신 코드처럼 객체안에 배열이 있거나, 배열안에 또 객체가 있는 등의 구조에서는 한줄로 표기할 경우 구분하기가 매우 어려워요.

하나하나 경계를 찾으면서 읽어야하기 때문에 조금만 복잡해져도 실수할 확률이 높아져요.

코드를 읽는 컴퓨터 입장에서는 상관이 없지만 우리가 프로그래밍 “언어” 라고 하는 이유는 말그대로 개발자가 소통하기 위해서라고 생각해요.

한줄 한줄 코드를 작성하실 때 항상 가독성에 대해 고민하는 습관을 가지면 좋을 것 같아요 ;)